### ISSUE TITLE
User Attributes entity is alternately broad and specific

### ISSUE THREAD

**Issue by pbchase** [ctsit/redcap_oncore_client#19]
The User Attributes entity is alternately broad and specific. The entity goes by these monikers in the ExternalModules.php

```
        $types['user_attributes'] = [
            'label' => 'Local User Information',
            'label_plural' => 'Local Users Information',
```

Yet for the plugin page (in config.json) we refer to it with _Enter Study Staff_.  i.e.

```
                "name": "Enter Study Staff",
                "icon": "group_gear",
                "url": "plugins/user_attributes.php"
```
Are these "study staff" details or are we defining a large list of user attributes? I am conflicted here. At UF, the study staff ID is the local UFID. It could be useful in other modules. Yet we are collecting it only within the context of the OnCore Client. 'user_attributes' feels more like system-wide service. 

My inclination is to rename this entity _study_staff_attributes_ or _oncore_study_staff_attributes_.  I welcome input on this matter.

**Comment by ChemiKyle** [comment]
If there's a possibility of using the `user_attributes` table later with other plugins maybe we should name it `institution_study_staff_identification`, as it only contains the REDCap username and their institution ID. It's currently not clear that the OnCore provided information is OnCore specific, perhaps `protocol_staff` should be changed to `oncore_protocol_staff_attributes`.

Below is what each one contains so you don't have to go back to look for them:  
Note: `staff_id` is the Institution ID, so maybe that should be changed as well.

```
        $types['user_attributes'] = [
            'label' => 'Local User Information',
            'label_plural' => 'Local Users Information',
            'properties' => [
                'staff_id' => [
                    'name' => 'Staff ID',
                    'type' => 'text',
                    'required' => true,
                ],
                'user_id' => [
                    'name' => 'User',
                    'type' => 'user',
                ],
            ],
        ];

        $types['protocol_staff'] = [
            'label' => 'OnCore Protocol Staff Information',
            'label_plural' => 'Local Users Information', // This is a typo
            'properties' => [
                'protocol_no' => [
                    'name' => 'Protocol Number',
                    'type' => 'text',
                    'required' => true,
                ],
                'staff_id' => [
                    'name' => 'Staff ID',
                    'type' => 'text',
                ],
                'stop_date' => [
                    'name' => 'Stop Date',
                    'type' => 'date',
                ],
            ],
        ];
```

**Comment by pbchase** [comment]
@ChemiKyle and I discussed this.  The new names will be:

- protocol_staff -> oncore_ protocol_staff
- user_attributes -> oncore_staff_identifier

**Comment by ChemiKyle** [comment]
Resolved by #26
